### ISSUE TITLE
No Column Name in CSV

### ISSUE THREAD

**Issue by jersearls** [jiananlu/faked_csv#2]
CSVs generated do not include column header name. Any idea why not?

**Comment by jersearls** [comment]
This functionality is intact in v.0.1.4. Code changes in v.0.1.5 appear to have broken this feature

**Comment by vjdavid** [comment]
I agree with @jersearls  the example on the readme page, seems working fine. And yes the headers of each column does not appear!
